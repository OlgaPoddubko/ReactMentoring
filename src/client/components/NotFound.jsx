import React from 'react';

const NotFound = props => (
    <div className="not-found">Sorry, page not found.
      <style jsx>{`
        .not-found {
          max-width: 1130px;
          margin: 0 auto;
          padding: 80px 35px;
          text-align: center;
          font-weight: bold;
          font-size: 48px;
        }
      `}</style>
    </div>
);

export default NotFound;
