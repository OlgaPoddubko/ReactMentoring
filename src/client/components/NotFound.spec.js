import React from 'react';
import NotFound from './NotFound';

describe('render NotFound', () => {
  it('should render NotFound', () => {
    const wrapper = shallow(<NotFound />);
    expect(wrapper).toMatchSnapshot();
  });
});
