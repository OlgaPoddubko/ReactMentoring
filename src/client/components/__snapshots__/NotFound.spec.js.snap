// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render NotFound should render NotFound 1`] = `
<div
  className="not-found"
>
  Sorry, page not found.
  <style
    jsx={true}
  >
    
        .not-found {
          max-width: 1130px;
          margin: 0 auto;
          padding: 80px 35px;
          text-align: center;
          font-weight: bold;
          font-size: 48px;
        }
      
  </style>
</div>
`;
